tensorbay/cli/utility.py

```diff
@@ -61,16 +62,14 @@ def read_config(config_filepath: str, profile_name: str) -> Tuple[str, str]:
     if not os.path.exists(config_filepath):
         click.echo(
             f"{config_filepath} not exist"
-            "\n\nPlease use 'gas config <accessKey>' to create config file",
+            "\n\nPlease use 'gas auth' or 'gas config' to create config file",
```

Comment:
if there is no `.gasconfig` file, that meanings there is no accesskey. So that the instruction should be only `gas auth`, and remove the `gas confg`